internal/service.binding/v1alpha2/service_binding.go

```diff
@@ -92,43 +91,25 @@ type ServiceBindingSpec struct {
 	Mappings []ServiceBindingMapping `json:"mappings,omitempty"`
 }
 
-// ServiceBindingConditionType is a valid value for ServiceBindingCondition.Type
-type ServiceBindingConditionType string
-
 // These are valid conditions of ServiceBinding.
 const (
-	// ServiceBindingReady means the ServiceBinding has projected the ProvisionedService secret and the Pod is ready to
-	// start
-	ServiceBindingReady ServiceBindingConditionType = "Ready"
+	// ServiceBindingReady means the ServiceBinding has projected the ProvisionedService
+	// secret and the Pod is ready to start
+	ServiceBindingConditionReady = "Ready"
 
-	// ServiceBindingProjectionReady means the ServiceBindingProjection has projected the ProvisionedService secret and
-	// the Pod is ready to start.  Extension.
-	ServiceBindingProjectionReady ServiceBindingConditionType = "ProjectionReady"
+	// ServiceBindingConditionProjectionReady means the ServiceBindingProjection has projected
+	// the ProvisionedService secret and the Pod is ready to start.  Extension.
+	ServiceBindingConditionProjectionReady = "ProjectionReady"
```

Comment:
is this something that PR #143 would remove?

Reply:
yea, I'll rebase this after that merges

Reply:
@arthurdm rebased